tonic-health/build.rs

```diff
@@ -1,5 +1,11 @@
+use std::env;
+use std::path::PathBuf;
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let grpc_health_v1_descriptor_set_path: PathBuf =
+        PathBuf::from(env::var("OUT_DIR").unwrap()).join("grpc_health_v1.bin");
```

Comment:
OUT_DIR might not be set except in the build environment.

Reply:
Pretty sure it is set when build.rs is ran.

Reply:
```
OUT_DIR — If the package has a build script, this is set to the folder where the build script should place its output. See below for more information. (Only set during compilation.)
```
 Found here: https://doc.rust-lang.org/cargo/reference/environment-variables.html#:~:text=OUT_DIR%20%E2%80%94%20If%20the%20package%20has,(Only%20set%20during%20compilation.)

Reply:
But, if you want, I can add a check instead of using .unwrap() here.

Reply:
My only question in that case would be, what behavior would you expect if the OUT_DIR didn't exist?

Would there just be an error saying that the OUT_DIR wasn't set and not produce the descriptor set?

What would say about this @LucioFranco?

Reply:
I think we can trust that `OUT_DIR` will be there, because this script is also compiling the proto I think its safe.